interfaces/builtin/basedeclaration.go

```diff
@@ -441,6 +441,11 @@ slots:
     deny-auto-connection: true
     deny-connection:
       on-classic: false
+  online-accounts-service:
+    allow-installation:
+      slot-snap-type:
+        - app
+    deny-connection: true
```

Comment:
The base declaration is fine since this is a trusted helper that is safe for all snaps to auto-connect.